src/zero.ts

```diff
@@ -0,0 +1,280 @@
+    description: '',
+    publisher: {
+        id: 0,
+        name: '',
+        apps: []
+    },
+    official: false,
+    public: false,
+    source_url: ''
+};
+
+const functionx = {
+    installation_id: 0,
+    type: ''
+};
+
+const emptyFile = {
+    hash: '',
+    name: '',
+    mime: '',
+    installation_id: 0,
+    organization_id: 0
+};
+
+const edgeAppInstance = {
+    app_id: 0,
+    installation_id: 0,
+    version: '',
+    config: {},
+    name: ''
+};
+
+const installationInfo = {
+    id: 0,
+    name: '',
+    client_id: 0,
+    organization_id: 0,
+    capabilities: [],
+    assigned: false
+};
+
+const installation = {
+    name: '',
+    organization_id: 0,
+    notes: '',
+    users: []
+};
+
+const logEntry = {
+    client_id: 0,
+    installation_id: 0,
+    timestamp: 0,
+    topic: '',
+    value: 0,
+    message: '',
+};
+
+const notificationMessage = {
+    installation_id: 0,
+    name: '',
+    text: ''
+};
+
+const notificationOutput = {
+    installation_id: 0,
+    name: '',
+    notification_message_id: 0,
+    notification_output_executor_id: 0,
+    config: {},
+};
+
+const notificationOutputExecutor = {
+    type: '',
+    name: '',
+    organization_id: 0,
+    config: {},
+    secret: undefined
+};
+
+const role = {
+    name: '',
+    priority: 0,
+    permissions: {}
+};
+
+const schedule = {
+    installation_id: 0,
+    executor: '',
+    day_of_week: '',
+    day_of_month: '',
+    month: '',
+    hour: '',
+    minute: '',
+    topic: '',
+    value: ''
+};
+
+const token = {
+    expire: 0,
+    session_timeout: 0,
+    name: '',
+    permissions: []
+};
+
+const zeroToken = {
+    type: '',
+    created: 0,
+    token: '',
+    accessed: [{
+        ip: '',
+        agent: '',
+        last_accessed: 0,
+    }],
+    last_used: 0,
+    current: false,
+    agent: ''
+};
+
+const topicBlacklist = {
+    topic: '',
+    mode: '',
+    enabled: false,
+    reason: '',
+    count: 0
+};
+
+const trace = {
+    id: 0,
+    path: '',
+    method: '',
+    timestamp: 0,
+    user_id: 0,
+    action: TraceAction.Create,
+    object_type: TraceObjectType.Installation,
+    object_id: 0,
+    description: ''
+};
+
+const user = {
+    email: '',
+    password: '',
+    first_name: '',
+    last_name: '',
+    role: 0,
+    sms_login: false,
+    mobile: '',
+    note: '',
+    organisations: [],
+    assigned_installations: [],
+    address,
+    expire_at: 0
+};
+
+const gatewayInformation = {
+    client_id: 0,
+    installation_id: 0,
+    api: '',
+    aam: '',
+    mqtt_broker: '',
+    mqtt_username: '',
+    mqtt_password: '',
+    mqtt_client_id: '',
+    timezone: ''
+};
+
+const organizationSimple = {
+    id: 0,
+    name: '',
+    parent: ''
+};
+
+export const zero = {
+    getEmptyDevicex: (): EmptyDevicex => clone({...devicex, ...emptyWithMeta}),
+    getZeroEmptyDevicex: (): Devicex => clone({...devicex, ...emptyWithMeta, ...emptyIdentifier, ...emptyCreationDate}),
```

Comment:
These getter should be named get<ModelName>. That is `getDevicex` here. Zero is already implied with the exported model name `zero`.